input/docs/fundamentals/preprocessor-directives.md

```diff
@@ -183,3 +183,20 @@ To check if symbol is defined, you have to use directives `#if` ... `#else` ...
     // code in this branch will be skipped
 #endif
 ```
+
+## Predefined symbols
+
+Some symbols can be predefined by Cake depending on the environment.
+
+| Symbol Name      | Description                                                    |
+| ---------------- | -------------------------------------------------------------- |
+| `CAKE`           | Always defined for code executed by Cake.                      |
+| `NETCOREAPP`     | Defined when the CLR is .NET Core                              |
+| `NETFRAMEWORK`   | Defined when the CLR is .NET Framework                         |
+| `NET461`         | Defined when the runtime framework is .NET Framework 4.6.1     |
+| `NETCOREAPP2_0`  | Defined when the runtime framework is .NET Core 2.0            |
+| `NETCOREAPP2_1`  | Defined when the runtime framework is .NET Core 2.1            |
+| `NETCOREAPP2_2`  | Defined when the runtime framework is .NET Core 2.2            |
+| `NETCOREAPP3_0`  | Defined when the runtime framework is .NET Core 3.0            |
+| `NETCOREAPP3_1`  | Defined when the runtime framework is .NET Core 3.1            |
+| `NETSTANDARD2_0` | Defined when the runtime framework is .NET Standard 2.0        |
```

Comment:
`NETSTANDARD2_0` is returned when .NET version could not be detected, with the assumption that all versions support NET Standard 2.0. I would suggest to change description to something like:

```
Defined when the runtime could not be determined but is .NET Standard 2.0 compatible.
```

Reply:
We've got NET5_0 too in develop.

Reply:
I've fixed the description for .NET Standard 2.0. Regarding NET 5.0 @devlead  i'm not sure how to proceed. I guess it all depends on when the new documentation will be published in relation to when the .NET 5.0 support will be released. If it already is released I can add it to the documentation right away.

Reply:
@mgnslndh I think we can go ahead and add in the new one, but perhaps we need an additional column in the table, or perhaps a footer with a note, saying that this is only available in 1.0.0+ of Cake?

Reply:
In general, an "available in" column specifying version might not be a bad pattern.

Reply:
Agree with an additional column mentioing when introduced. For me it would be also be OK to merge like this and add the additional column in a second PR. Whatever you prefer.

Reply:
Yip, either works for me.  @mgnslndh would you be up for a follow up PR to address what has been suggested here?

Reply:
Yes, can do that.